UI/src/sass/core.scss

```diff
@@ -5,3 +5,8 @@
 .display-pre-wrap {
   white-space: pre-wrap;
 }
+
+.feedback-iframe {
+  width: 100%;
+  height: 800px;
```

Comment:
Instead of setting the height here to `800px` (Absolute), can we make it `100vh` size relative to the viewport (browser window, basically) so it is essentially the same size as the max height for the iframe?

Reply:
Good shout - have made that change